lib/mix/lib/mix/tasks/compile.erlang.ex

```diff
@@ -74,7 +74,15 @@ defmodule Mix.Tasks.Compile.Erlang do
     end
 
     erlc_options =
-      erlc_options ++ [:debug_info, :return, :report, outdir: compile_path, i: include_path]
+      erlc_options ++
+        [
+          :debug_info,
+          :return,
+          :report,
+          :no_spawn_compiler_process,
```

Comment:
@josevalim is this the option you meant?

Reply:
Yes!

Reply:
But you want to set it only inside the parallel compilation... If it is not easy to do that, then we should remove this option.

Reply:
There is no nice way of doing it unfortunately. How about we set the option always and spawn and wait for processes serially for non-parallel compilation?

Reply:
Given we are compiling from a file and not from AST, setting this option is not really relevant. So it is fine to not include it.